MinecraftClient/MinecraftClient.csproj

```diff
@@ -154,6 +154,19 @@
     <Compile Include="Protocol\Handlers\Forge\FMLVersion.cs" />
     <Compile Include="Protocol\Handlers\PacketIncomingType.cs" />
     <Compile Include="Protocol\Handlers\PacketOutgoingType.cs" />
+    <Compile Include="Protocol\Handlers\PacketTypes\Palettes\PacketPalette111.cs" />

```

Comment:
To avoid too many directories and for consistency, it would be better to use the same architecture as other namespaces:

```
MinecraftClient.Protocol.Handlers
 +- PacketTypesIn.cs
 +- PacketTypesOut.cs
 +- PacketTypeHandler.cs (or replace Protocol18PacketTypes.cs with it)
MinecraftClient.Protocol.Handlers.PacketPalettes
 +- PacketPalette111.cs
 +- [...]
```

Reply:
Then I am going to delete the old implementation of packet type.